spec/models/patron_spec.rb

```diff
@@ -51,4 +69,33 @@
       expect(patron.email).to be_nil
     end
   end
+
+  describe '#expired?' do
```

Comment:
We should probably have additional tests for the other logic branches:

a) a privileges date in the future
b) no privileges date